backend/infrahub/support/macro.py

```diff
@@ -58,7 +58,8 @@ def _parse_template(self) -> nodes.Template:
 
     def render(self, variables: dict[str, Any]) -> str:
         template = self.env.from_string(self.macro)
-        return template.render(variables)
+        output_dict = {key: ("" if value is None else value) for key, value in variables.items()}
+        return template.render(output_dict)
```

Comment:
This is to ensure that we use a blank string instead of `None` if a given value would be missing.

Reply:
We could still use None and do a default(omit) in the Jin’s depending if the use case

Reply:
I reverted this part for now.